src/types.rs

```diff
@@ -196,6 +196,45 @@ pub struct GetMempoolInfo {
     pub unbroadcastcount: usize,
 }
 
+/// Response from `getrawmempool` with `verbose=true`.
+///
+/// The top-level map key is the txid, and the value contains detailed mempool info per tx.
+pub type GetRawMempoolVerbose = BTreeMap<Txid, MempoolEntry>;
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize)]
+pub struct MempoolEntry {
+    pub vsize: usize,
+    pub weight: usize,
+    #[serde(default)]
+    pub fee: Option<f64>, // DEPRECATED
```

Comment:
NACK use `deserialize_bitcoin`:

```suggestion
    #[serde(default, deserialize_with = "deserialize_bitcoin")]
    pub fee: Option<f64>, // DEPRECATED
```

Also I don't see this as deprecated in https://bitcoincore.org/en/doc/29.0.0/rpc/blockchain/getrawmempool/

Reply:
IIRC its been replaced by `fees` in 29.0 or perhaps an earlier version. Happy to just remove this. I included a deprecation comment to mirror what is done on the [core rpc types crate](https://docs.rs/corepc-types/latest/src/corepc_types/v17/blockchain/mod.rs.html#494)